docs/source/pythonapi/examples/mdgxs-part-ii.ipynb

```diff
@@ -822,178 +822,27 @@
    },
    "outputs": [
     {
-     "data": {
```

Comment:
:-( no you arent missing anything. I didnt realize my failed completion of mdgxs-part-ii (discussed elsewhere in this PR) was saved.  I've got the copy from the develop branch back in now.